docs/docker/Dockerfile.bionic

```diff
@@ -72,3 +63,16 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 
 # 9. Run everything after as non-privileged user.
 USER pwuser
+
+# === BAKE BROWSERS INTO IMAGE ===
+
+# 1. Add tip-of-tree Playwright package to install its browsers.
+#    The package should be built beforehand from tip-of-tree Playwright.
+COPY --chown=pwuser ./playwright.tar.gz /home/pwuser/playwright.tar.gz
+
+# 2. Install playwright and then delete the installation.
+#    Browsers will remain downloaded in `/home/pwuser/.cache/ms-playwright`.
+RUN mkdir /home/pwuser/tmp && cd /home/pwuser/tmp && npm init -y && \
+    npm i ../playwright.tar.gz && \
+    cd ../ && rm -rf tmp && rm /home/pwuser/playwright.tar.gz
```

Comment:
Is there a reason not to install playwright itself along with the browsers?

Reply:
there's a way to install package globally: https://nodejs.org/api/modules.html#modules_loading_from_the_global_folders
However, global modules are highly discouraged in node.js - so I'd refrain from this, at least for now.